### DISCUSSION TITLE
Move Playhead to Prev/Next Segment

### DISCUSSION THREAD

**Discussion by robinmattern** [mifi/lossless-cut#1310] [General]
1. I thought {UP}/{DOWN} might do this, but it only jumps to the segment in the list on the left.  It doesn't move the playhead.  It does highlight the segment in the timeline, but if you are zoomed in you don't see it. 
2. I thought {SHIFT-LEFT}/{SHIFT-RIGHT} might do this, but it only moves the playhead once.  Holding it down does nothing.  
3. I found that this works: {UP}{SHIFT-LEFT} moves the playhead to the start of the previous segment, and {DOWN}{SHIFT-RIGHT} moves the playhead to the end of the next segment.  This latter move is not very intuitive while playing the video, but if you reverse the keystrokes to {SHIFT-RIGHT}{DOWN}, then it does play from one segment to the next.  You can't hold these two keystrokes down, but you can get a good rhythm going.  

Maybe I'm missing something, but this seems to be a very common use case: to quickly play the beginning of each segment.   Since holding the {SHIFT-RIGHT} key combination down does nothing, it might be reasonable to implement moving the playhead continuously from one segment to the next.

**Comment by mifi** [comment]
I agree this is a nice keyboard shortcut to have. I'll create a feature request
